load/load.py

```diff
@@ -0,0 +1,75 @@
+import os
+import zipfile
+import duckdb
+import tempfile
+from datetime import datetime
+
+# Step 1: Extract the zip file into a temporary directory
+def extract_zip_file(zip_file_path):
```

Comment:
Currently this function returns a temp_dir without cleaning it up, which leads to confusing behavior in the main function. To use this function properly, you need to know that this function has a "side-effect" (meaning an unwanted side-effect behavior) of creating a temp folder without destroying it.

A better way woudl be to pass the target directory as second argument to this function, and let the main function caller handle creating and cleaning up the temp_dir there in the same context.

Reply:
done